lib/controller/page/type/page.summary/page.summary.controller.js

```diff
@@ -12,7 +12,11 @@ const CONSTANTS = require('../../../../constants/constants')
 const {
   SERVICE_OUTPUT_EMAIL,
   SERVICE_OUTPUT_JSON_ENDPOINT,
-  SERVICE_OUTPUT_JSON_KEY
+  SERVICE_OUTPUT_JSON_KEY,
+  SERVICE_SLUG,
+  PLATFORM_ENV,
+  DEPLOYMENT_ENV,
+  PORT
```

Comment:
is this the correct PORT to use? or hardcode to 3000?

Reply:
`PORT` is the correct port to use, yes